core-bundle/src/Csrf/MemoryTokenStorage.php

```diff
@@ -15,6 +15,9 @@
 use Symfony\Component\Security\Csrf\Exception\TokenNotFoundException;
 use Symfony\Component\Security\Csrf\TokenStorage\TokenStorageInterface;
 
+/**
+ * @deprecated since Contao 4.8, to be removed in Contao 5.0
+ */
 class MemoryTokenStorage implements TokenStorageInterface
```

Comment:
I don't think we need to keep that, really. If we remove the service definition too, there's no point in still providing that? It's internal stuff anyway, isn't it? Why would anyone use that in their own code?

Reply:
I doesn’t hurt and improves BC so we should keep it I think. And I should probably add the add the `contao.csrf.token_storage` service back.

Reply:
If we decide to keep BC here, we need to restore the `contao.csrf.token_storage` service. I share @Toflar's view though.